src/components/forms/VsiForm.js

```diff
@@ -55,21 +55,26 @@ class VsiForm extends Component {
 
```

Comment:
cant comment on the line but handleInputChange is where `subnet` is getting added unncessarily, add a check for if `isTeleport` and from there either add `subnet` or `subnets` to the obj that goes into state